.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+### Description
+
+Fixes #[ISSUE NUMBER]
+
+Replace this line by a one sentence summary of your PR.
+
+If necessary, use the following space to provide context or more details.
+
+### Contribution Checklist
+
+If your contribution modifies code in the core library (not docs, tests, or examples), please fill the following checklist.
+
+- [x] My contribution modifies code in the main library.
+- [ ] My modifications are tested.
+- [ ] My modifications are documented.
+
+Please copy-paste the output of running `make alltests` below.
```

Comment:
My development flow doesn't really require me to do `make alltests`, probably we could remove this part and replace with additional comments or something... YKWIM?